README.md

```diff
@@ -208,6 +208,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Abstract API](https://www.abstractapi.com) — API suite for a variety of use cases including IP geolocation, gender detection or even email validation.
   * [Apify](https://www.apify.com/) — Web scraping and automation platform to create an API for any website and extract data. Ready-made scrapers, integrated proxies, and custom solutions. Free plan with $5 platform credits included every month.
   * [API Mocha](https://apimocha.com) - Completely free online API mocking for testing and prototyping.  Make up to 500 requests per day, fully customizable API responses, download mock rules as a Postman collection.
+ * [API Security Academy](https://escape.tech/academy/) - A free and interactive platform dedicated to learning how to fix vulnerabilities in GraphQL applications — all directly in your browser. 
```

Comment:
not sure this is really suited to the list, its kind of just like soneone should have written a document and so being single tech specific I think there are better places to add it.

Not saying its useful, its just not a SaaS.